ckanext/data_qld/plugin.py

```diff
@@ -187,6 +191,41 @@ def get_blueprint(self):
             })
         return blueprint
 
+    # IQA
+    def custom_resource_score(self, resource, resource_score):
+        # If resource score is 3 and is of CSV format
+        if resource_score.get('openness_score', 0) == 3 and resource_score.get('format', '').upper() == 'CSV':
+            # If resource has a JSON schema which validated , set score to 4
+            if hasattr(resource, 'extras') and resource.extras.get('schema', None) and resource.extras.get('validation_status', None) == 'success':
+                resource_score['openness_score'] = 4
+                resource_score['openness_score_reason'] = toolkit._('Content of file appeared to be format "{0}" which receives openness score: {1}.'
+                                                                    .format(resource_score.get('format', ''), resource_score.get('openness_score', '')))
+
+        return resource_score
+
+    # IReport
+    def register_reports(self):
+        """Register details of an extension's reports"""
+        from ckanext.qa import reports
+        try:
+            from collections import OrderedDict  # from python 2.7
+        except ImportError:
+            from sqlalchemy.util import OrderedDict
+
+        data_usability_rating_info = {
+            'name': 'data_usability_rating',
+            'title': toolkit._('Data usability rating'),
+            'description': toolkit._("Datasets rated according to rules similar to those used by the Australian Government's Linked Data Rating derived from (but different to) Tim Berners-Lee's Five Stars of Openness."),
```

Comment:
This line is very long and requires scrolling to read it all on Github.

Reply:
@ThrawnCA sounds like you're using Unified view in Github. switch to Split view and it will wrap so you can read it without scrolling.

Reply:
Extremely long lines are still frowned on.

Reply:
This code was removed in another commit, I have merged the changes.